README.md

```diff
@@ -1,101 +1,84 @@
-ensure the framework you use will be installed, avoiding the long install times
-and possible version requirements issues installing unused frameworks may bring.
+With the model tracked by Git-Theta, you can treat the model checkpoint exactly like you would any other file in a Git repository. All of the regular Git commands like `add`, `commit`, `push`, `pull`, `checkout`, `status`, `diff`, etc. will work on your model the way they would on any other file.
 
-For example, install git-theta with only pytorch checkpoint support:
+Additionally, when staging a change to a model, you can provide Git-Theta with additional information about ***what type*** of change is being staged (e.g., a sparse update, a low-rank update, etc.) by running `git theta add model.pt --update-type <update type>`. This allows Git-Theta to store the model update more efficiently, saving disk space and bandwidth when `push`-ing or `pull`-ing.
 
-``` bash
-cd git-theta
-pip install .[pytorch]
-```
+# Why is this better than using Git or Git LFS?
+Git on its own can certainly be used for versioning non-text files like model checkpoints. However, the main limiting factors are that
+1. Git remotes like Github and Bitbucket have a maximum file size (~50MB)
+2. Git is not designed to handle very large repositories
 
-If you already have your framework of choice installed (i.e. pip doesn't need
-to ensure it is installed), you can just install git-theta with `pip install .`
+There are a number of existing solutions for storing large files with Git that circumvent the maximum file and repository size, such as Git LFS. These work by pushing large files to an external LFS endpoint rather than the Git remote. The main issue with using Git LFS-like systems for versioning ML models is that they are unaware of the structure of ML models. 
 
-## Initializing git theta
-Initialize git theta by running:
-```bash
-git theta install
-```
+Imagine you have a checkpoint that you are updating by [training only a small percentage of the parameters](https://arxiv.org/abs/2111.09839), [training only a few of the layers](https://arxiv.org/abs/2106.10199), or by [adding new trainable modules](https://arxiv.org/abs/1902.00751). In these cases, most of the model remains the same and only a small fraction of the model gets modified. However, tools like Git LFS just see that the checkpoint file has changed, and will store the new version of the checkpoint file in its entirety. 
 
-The following lines will be added to `~/.gitconfig` after successful installation.
-```
-[filter "lfs"]
-        smudge = git-lfs smudge -- %f
-        required = true
-        clean = git-lfs clean -- %f
-[filter "theta"]
-        clean = git-theta-filter clean %f
-        smudge = git-theta-filter smudge %f
-        required = true
-```
+Git-Theta is aware of the structure of ML models and is designed to store only the parts of a model that have changed from its previous version as efficiently as possible.
```

Comment:
`Git-Theta is aware of the structure of ML models`

I might be being too pedantic but I think we should reword to something like `ML models are comprised for multiple parameter groups and Git-Theta is designed to store...`

Because we don't actually know the structure (like if you are reusing embedding for not, etc) we just know it is made of multiple parameters that can change ~independently